www/android/geolocation.js

```diff
@@ -52,7 +62,20 @@ module.exports = {
           var geo = cordova.require('cordova/modulemapper').getOriginalSymbol(window, 'navigator.geolocation');
           geo.clearWatch(args[0]);
         }
-        exec(win, error, "Geolocation", "getPermission", []);
+        var fail = function() {
+            if (error) {
+                error(illegalAccessError);
+            }
+        };
+        exec(win, fail, "Geolocation", "getPermission", []);
     }
 };
 
+// mimic PositionError to be compliant with W3C spec
+var illegalAccessError = {
```

Comment:
Currently PositionError is not available on Andoroid as we don't include that file, but I think we can include it and use it - I'll update PR.